nn_sdwan/manifest.json

```
@@ -0,0 +1,44 @@
+{
+  "display_name": "Netnology SD-WAN",
+  "maintainer": "info@netnology.io",
+  "manifest_version": "1.0.0",
+  "name": "nn_sdwan",
+  "metric_prefix": "nn_sdwan.",
+  "metric_to_check": [
+    "nn_sdwan.device_control_status",
+    "nn_sdwan.app_aware_routing.latency"
+  ],
+  "creates_events": false,
+  "short_description": "SDWAN Controller Metric Exporter by Netnology",
+  "guid": "a7c79ba9-a4b7-4fa4-b4aa-3868458e2101",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Netnology SD-WAN Integration",
```

```suggestion
  "public_title": "Datadog-Netnology SD-WAN",
```
Non-blocking comment, if you did want to the shorten the name. It looks like you can override this value to fix the name, similar to windows performance counters: https://github.com/DataDog/integrations-core/blob/master/windows_performance_counters/manifest.json#L15